app/assets/stylesheets/components/_modal.scss

```diff
@@ -42,8 +43,8 @@
   .modal-content {
     position: relative;
     flex: 1;
-    padding: $box-space;
     overflow: auto;
+    max-height: 85vh;
```

Comment:
I'm ok with this in modern desktop browsers, but it will give us some problems in Safari iOS. So we need to remember it when we will make the responsive.